### PR TITLE
feat: add timeout params into config

### LINKED ISSUE #33: Add timeout params for API server into config.toml
```
[api]
listen_addr =
write_timeout
read_timeout
```

and set the write timeout long enough

### PR DESCRIPTION
**Pull request by audtlr24**
closes #33.

- The underscore has been unified into a hyphen form.

### CODE REVIEW

File: config/toml.go [hunk 1]
```diff
@@ -23,12 +23,12 @@ log-level = "{{ .BaseConfig.LogLevel }}"
 oracle-mnemonic = "{{ .BaseConfig.OracleMnemonic }}"
 oracle-acc-num = "{{ .BaseConfig.OracleAccNum }}"
 oracle-acc-index = "{{ .BaseConfig.OracleAccIndex }}"
-listen_addr = "{{ .BaseConfig.ListenAddr }}"
-data_dir = "{{ .BaseConfig.DataDir }}"
+listen-addr = "{{ .BaseConfig.ListenAddr }}"
```
Comment (inchori) [inline config/toml.go:26]: Does we need to move `listen-addr` in API Configuration or just leave in here?

**Review by inchori — APPROVED** [review]
LGTM
Left one minor comment.

File: config/toml.go (hunk 1 above)
Comment (audtlr24) [reply config/toml.go:26]: I think `listen-addr` is only used in API. It would be better to move it. Thanks.

**Review by gyuguen — APPROVED** [review]
LGTM

**Review by H4NLee — APPROVED** [review]
lgtm

**Pull request merged**